frontend/src/components/DrawerButton.tsx

```diff
@@ -0,0 +1,49 @@
+import { SvgIconComponent } from "@mui/icons-material";
+import {
+  ButtonProps,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+} from "@mui/material";
+
+type DrawerButtonProps = ButtonProps & {
+  Icon: SvgIconComponent;
+  buttonDescription: string;
+};
+
+export const DrawerButton = ({
+  Icon,
+  buttonDescription,
+}: DrawerButtonProps) => {
+  return (
+    <ListItemButton
+      sx={{
+        display: "grid",
+        justifyContent: "center",
+        "&:hover": {
+          background: "none",
+        },
```

Comment:
By gestalt principles of visual perception, groupings of buttons close to one another with similar colouring (Blue) are sufficient to associate like items. I feel that that is a sufficient form of visual cue to indicate that the buttons can be used. Hence I actively removed a lot of hover attributes from the app

Reply:
I think we can add the hovering. Was thinking that by gestalt principles, since we are already grouping / distinguishing the buttons by using a distinct color (blue), it will be sufficient visual cue to indicate that its a clickable button